src/test/java/com/puppycrawl/tools/checkstyle/PackageNamesLoaderTest.java

```diff
@@ -82,28 +81,34 @@ public void testDefault()
                 packageNames.size());
     }
 
+    /**
+     * Tests the loading of package names. This test can be done without mocking.
+     * Put {@literal checkstyle_packages.xml} in test resources will be sufficient.
```

Comment:
It seems this sentence is not a complete thought. You also shouldn't start the next sentence with `But`.
Why don't we combine the 2, and explain what we are not `putting in checkstyle_packages.xml`.

How about something like this for the 3 sentences:
`This test needs mocking, as XXX would have to be placed in {@literal checkstyle_packages.xml}, but this will affect every test, which is undesired.`
Replace `XXX` with whatever we are mocking into the file.

Reply:
done. Thanks for the suggestion.